### PR TITLE
Improving neo4j.conf with comments about core edge settings.

### PR DESCRIPTION
**Pull request by maxsumrall**

### CODE REVIEW

File: packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf [hunk 1]
```diff
@@ -193,23 +194,95 @@ ha.pull_interval=10
...
+
+# Database mode
+# Allowed values:
+# CORE - Core member of the cluster, part of the consensus quorum.
+# EDGE - Edge member of the cluster, an eventually-consistent read-only instance of the database.
+# To operate this Neo4j instance in Core/Edge mode as a core member, uncomment this line:
+#dbms.mode=CORE
+
+#The expected size of the core cluster.
+core_edge.expected_core_cluster_size=3
```
Comment (davidegrohmann) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf]: shouldn't all code_edge settings commented out?

File: packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf [hunk 2]
```diff
@@ -154,8 +155,8 @@ dbms.connector.https.address=localhost:#{default.https.port}
 # must be the configured IP address for one of the local interfaces.
 #ha.host.data=127.0.0.1:6001
 
-# The interval at which slaves will pull updates from the master. Comment out
-# the option to disable periodic pulling of updates. Unit is seconds.
+# The interval, in seconds, at which slaves will pull updates from the master. You must comment out
+# the option to disable periodic pulling of updates.
 ha.pull_interval=10
```
Comment (davidegrohmann) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf]: it looks like this should be commented out...

Comment (jimwebber) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf]: I think it means the end user must comment it out if they don't want to poll. Leave it as is?

File: packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf (hunk 1 above)
Comment (jimwebber) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf]: Yes.

File: packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf (hunk 2 above)
Comment (davidegrohmann) [inline packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf]: Fine but by default HA mode is disabled: https://github.com/maxsumrall/neo4j/blob/f4fc002ef31ad14d16a78e0beb4a09cccf37a906/packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf#L137

**Pull request merged**